L1Trigger/Configuration/python/SimL1Emulator_cff.py

```diff
@@ -49,10 +49,10 @@
 # Next we load ES producers for any conditions that are not yet in GT,
 # using the Era configuration.
 #
```

Comment:
Add line with
`if stage1L1Trigger.isChosen():`
then, uncomment and indent the three lines below.

This will allow Stage1 emulator to be configured from static files,
and everything that came after will be configured from DB.

Reply:
or better use,
`if not stage2L1Trigger.isChosen():`

Reply:
done